build.gradle

```diff
@@ -17,40 +20,46 @@ apply plugin: 'com.android.library'
 apply plugin: 'com.f2prateek.javafmt'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion '25.0.2'
+    compileSdkVersion 28
+    buildToolsVersion '28.0.2'
 
     defaultConfig {
         minSdkVersion 14
-        targetSdkVersion 25
+        targetSdkVersion 28
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_7
-        targetCompatibility JavaVersion.VERSION_1_7
+        sourceCompatibility JavaVersion.VERSION_1_8
```

Comment:
I think we should still target 1.7. Otherwise we may accidentally use 1.8 features, which would break the SDK for customers using JDK 1.7.

Reply:
Good point. Currently we are not using any Java 8 feature and Android studio will detect that, but it's safer to keep targeting 1.7